h/templates/panels/group_invite.html.jinja2

```diff
@@ -5,8 +5,7 @@ Sharing the link below lets people join this group:
 <div class="group-invite__container js-copy-button">
   <input class="group-invite__input js-select-onfocus"
          data-ref="input"
-         value="{{ group_url }}"
-         readonly>
+         value="{{ group_url }}">
```

Comment:
Yes indeed, although I think it is better that the user gets a form input that they can accidentally edit than one which they can't copy the text from at all (if scripts fail to load on Mobile Safari).